src/security/PeerOptions.cc

```diff
@@ -656,6 +657,8 @@ Security::PeerOptions::updateContextNpn(Security::ContextPointer &ctx)
 
 #if USE_OPENSSL && defined(TLSEXT_TYPE_next_proto_neg)
     SSL_CTX_set_next_proto_select_cb(ctx.get(), &ssl_next_proto_cb, nullptr);
+#else
+    (void)ctx;
 #endif
 
     // NOTE: GnuTLS does not support the obsolete NPN extension.
```

Comment:
Please move this comment to the new `#else` clause it belongs to (unless you want to add a dedicated GnuTLS clause).

Reply:
done